### PR TITLE
Improve logs on ConsoleProxyManagerImpl and refactor a few process

### PR DESCRIPTION
**Pull request by GutoVeronezi**
### Description
This PR intends to improve logging in the class `ConsoleProxyManagerImpl` to facilitate troubleshooting and refactor a few process.

### Types of changes
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Enhancement (improves an existing feature and functioanlity)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale
- [ ] Major
- [x] Minor

### How Has This Been Tested?
It has been tested locally on a test lab.

### CODE REVIEW

File: server/src/main/java/com/cloud/consoleproxy/ConsoleProxyManagerImpl.java [hunk 1]
```diff
@@ -238,17 +238,20 @@
...
     private String _consoleProxyUrlDomain;
 
-    // global load picture at zone basis
     private SystemVmLoadScanner<Long> _loadScanner;
-    private Map<Long, ZoneHostInfo> _zoneHostInfoMap; // map <zone id, info about running host in zone>
-    private Map<Long, ConsoleProxyLoadInfo> _zoneProxyCountMap; // map <zone id, info about proxy VMs count in zone>
-    private Map<Long, ConsoleProxyLoadInfo> _zoneVmCountMap; // map <zone id, info about running VMs count in zone>
+    private Map<Long, ZoneHostInfo> _zoneHostInfoMap;
+    private Map<Long, ConsoleProxyLoadInfo> _zoneProxyCountMap;
+    private Map<Long, ConsoleProxyLoadInfo> _zoneVmCountMap;
```
Comment (DaanHoogland) [inline server/src/main/java/com/cloud/consoleproxy/ConsoleProxyManagerImpl.java:244]: while touching this anyway, we might want to remove the silly undescores.

**Review by DaanHoogland — APPROVED** [review]
clgtm

File: server/src/main/java/com/cloud/consoleproxy/ConsoleProxyManagerImpl.java (hunk 1 above)
Comment (GutoVeronezi) [reply server/src/main/java/com/cloud/consoleproxy/ConsoleProxyManagerImpl.java]: @DaanHoogland done. I removed all initial underscore of variables and renamed some. Also, as it was used in `ConsoleProxyManagerTest`, I changed it too.

Comment (DaanHoogland) [reply server/src/main/java/com/cloud/consoleproxy/ConsoleProxyManagerImpl.java]: you are taking a radical aproach to my comment. I applaud you for that :clap: .  I'll run the smoke tests again just to be sure.

**Comment by DaanHoogland** [comment]
rebuilding and re-testing I think these errors are environmental/incidental.
@GutoVeronezi can you find someone to review/test for a second :+1: , please?

**Comment by DaanHoogland** [comment]
@GutoVeronezi , can you also rebase and check if it builds?

**Comment by GutoVeronezi** [comment]
@blueorangutan package

**Comment by GutoVeronezi** [comment]
@DaanHoogland rebase done, waiting for test results.

**Comment by GutoVeronezi** [comment]
@DaanHoogland actually, there really was an error; It was fixed with last commit. 

Most of the checks passed now, those that failed seems to fail due to environmental reasons; They were restarted.

**Comment by GutoVeronezi** [comment]
Can anyone review this?

**Comment by rhtyd** [comment]
This will need manually testing novnc and old cpvm consoles.
@GutoVeronezi can you review the one failure - does not look like related to this PR though.
@blueorangutan package

**Comment by GutoVeronezi** [comment]
> @GutoVeronezi can you review the one failure - does not look like related to this PR though.

@rhtyd The failure refers to `The VLAN tag to use for new guest network, 2468 is already being used for dynamic vlan allocation for the guest network in zone pr4959-t919-kvm-centos7`.

**Review by GabrielBrascher — APPROVED** [review]
Code LGTM. I've did not test it though.

**Comment by GutoVeronezi** [comment]
@DaanHoogland @GabrielBrascher is there anything else to do?

**Comment by DaanHoogland** [comment]
trillian failures are not consistent and seem env related and not code. @borisstoyanov @vladimirpetrov could you test this, please?

**Comment by GutoVeronezi** [comment]
Any update about this?

**Comment by GutoVeronezi** [comment]
Merging it based on LGTMs ([here](https://github.com/apache/cloudstack/pull/4959#pullrequestreview-650227196) and [here](https://github.com/apache/cloudstack/pull/4959#pullrequestreview-687790964)) and packages and tests results ([here](https://github.com/apache/cloudstack/pull/4959#issuecomment-884010810) and [here](https://github.com/apache/cloudstack/pull/4959#issuecomment-885121592)).

**Pull request merged**